### DISCUSSION TITLE
Onboard camera

### DISCUSSION THREAD

**Discussion by telemetryf1** [theOehrly/Fast-F1#201] [Q&A]
Hi, I have another question. Is there a way to get video of onboard camera during a specific lap or an entire event? Thanks for your replies

**Comment by theOehrly** [comment]
No, that is not possible.

**Comment by walteij** [comment]
Not possible with FastF1, but if you have a F1TV subscription, you can re-watch races and onboards.
If you want to view multiple streams of the race live, [F1 Multiview](https://beta.f1mv.com/) is the best solution for now, it does however have a 30 seconds delay compared to TV feed.
You can select several streams and order them the way you like:
[A screenshot of Silverstone Free practice 1](https://cdn.discordapp.com/attachments/915183270034026556/994916942630625322/IMG_20220617_204110_hdr2.jpg)

[RaceControl](https://github.com/robvdpol/RaceControl) can be used to download the streams, but only after DRM has been removed by F1TV, usually 2 days after the race was done.

**Reply by telemetryf1** [reply]
Oh thanks
